feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -97,3 +106,7 @@ func (srv *CentralSessionController) TerminateSession(
 
 	return (&protos.UnimplementedCentralSessionControllerServer{}).TerminateSession(ctx, request)
 }
+
+func (srv *CentralSessionController) Close() {
```

Comment:
add a comment on this function mentioning this is to gracefully close CentralSessionController. If we ever have to add to add more tasks to close we will know where to add them

Also note that it would be good to have a sbi client that would take care of that instead of placing this on the n7 level